src/Serilog.HttpClient/MaskHelper.cs

```diff
@@ -52,7 +52,7 @@ public static string MaskFields(this string json, string[] blacklist, string mas
                 MaskFieldsFromJToken(jObject, blacklist, mask);
             }
 
-            return jsonObject.ToString();
+            return JsonConvert.SerializeObject(jsonObject);
```

Comment:
I switched to this. I _think_ it's a little faster than ToString and it doesn't format the json, so you save some whitespace